app/inject/ProductProvider.java

```diff
@@ -21,7 +22,7 @@
     private static final long ALLOWED_TIMEOUT = 3000;
 
     @Inject
-    public ProductProvider(final PlayJavaSphereClient playJavaSphereClient) {
+    public ProductProvider(final PlayJavaSphereClient playJavaSphereClient, final ImportService importService) {
```

Comment:
Can you add here a comment why you have a dependency to `ImportService` but don't use it?